ads/pubmine.md

```diff
@@ -55,3 +55,12 @@ Please note that the height parameter should be 15 greater than your ad size to
 - `data-section`: Pubmine slot identifier
 - `data-pt`: Enum value for page type
 - `data-ht`: Enum value for hosting type
+
+## Consent Support
+
+Pubmine's amp-ad adheres to a user's consent in the following ways:
+
+- `CONSENT_POLICY_STATE.SUFFICIENT`: Pubmine amp-ad will display a personalized ad to the user.
+- `CONSENT_POLICY_STATE.INSUFFICIENT`: Pubmine amp-ad will display a non-personalized ad to the user.
+- `CONSENT_POLICY_STATE.UNKNOWN_NOT_REQUIRED`: Pubmine amp-ad will display a personalized ad to the user.
+- `CONSENT_POLICY_STATE.UNKNOWN`: Pubmine amp-ad will display a non-personalized ad to the user.
```

Comment:
Question: I found that you introduced a new flag called `npaOnUnknownConsent`, does that mean the behavior here is customizable?

Reply:
I found that if the `data-npa-on-unknown-consent` attribute was set then it would throw an error that `npaOnUnknownConsent` wasn't a valid parameter. Maybe including it here is overkill, but I wanted to be sure we captured the possibility. Or, is there a better way to handle this?

Reply:
Yes `#validateData` does check that all `data-` attributes are supported. 

My assumption is that in the `data-npa-on-unknown-consent=true`  case, you'll display a non-personalized ad to the user. And in the `data-npa-on-unknown-consent=false`  case, you'll display a personalized ad to the user. Let me know if that's correct?

And if so, I'd recommend clarify the behavior here in the doc. Thanks.

Reply:
Just pushed an update to clarify `data-npa-on-unknown-consent` use and no `data-block-on-consent` attribute effect.